coders/heic.c

```diff
@@ -1114,9 +1117,9 @@ static MagickBooleanType WriteHEICImage(const ImageInfo *image_info,
     */
     if (lossless != MagickFalse)
       error=heif_encoder_set_lossless(heif_encoder,1);
-    else if (image_info->quality != UndefinedCompressionQuality)
+    else if (image->quality != UndefinedCompressionQuality)
```

Comment:
Please revert these `image_info` to `image` changes. We don't want to use `image->quality` here.

Reply:
ok, by the way, the quality value in image_info structure is zero now, did i missing something?